docker-compose.yml

```
@@ -0,0 +1,9 @@
+services:
+  simple-jade-pin-server:
```

```suggestion
  simplejadepinserver:
```
Avoid using `-` in YAML keys due to potential compatibility problems.

---

Changed from `-` to `_`